src/components/DownloadImageDialog/ApplicationInstructions.tsx

```diff
@@ -104,7 +107,13 @@ export const ApplicationInstructions = memo(function ApplicationInstructions({
 
 	const finalInstructions = [
 		'Use the form on the left to configure and download balenaOS for your new device.',
-		...interpolatedInstructions,
+		...(secureboot
+			? [
+					...interpolatedInstructions.slice(0, 6),
+					`${interpolatedInstructions[6].split('.').slice(0, 2)}.`,
```

Comment:
Isn't there a better way to detect this? Like find the instruction step that includes x/y/z?